### PR TITLE
Image as generic parameter

### PR DESCRIPTION
**Pull request by ReDetection**
I wasn't able to get rid of this extra spacing. Need help with it 

touch bar shot 2018-04-12 at 23 49 54

### CODE REVIEW

**Comment by Toxblh** [comment]
touch bar shot 2018-04-13 at 01 13 45

**Comment by Toxblh** [comment]
Also need to delete the staticImageButton from the instruction

File: MTMR/AppleScriptTouchBarItem.swift [hunk 1]
```diff
@@ -36,7 +36,7 @@ class AppleScriptTouchBarItem: CustomButtonTouchBarItem {
         print("refresh happened")
         let scriptResult = self.execute()
         DispatchQueue.main.async {
-            self.button.title = scriptResult
+            self.button.title = scriptResult.trim()
```
Comment (ReDetection) [inline MTMR/AppleScriptTouchBarItem.swift]: I don't think it's a good idea. This script probably made non-empty output for a reason – like in btt you may use empty string to remove button from the bar. So this trim will silently remove the button, which might be unintended. Let's allow script developer to decide, ok?

File: MTMR/TouchBarController.swift [hunk 2]
```diff
@@ -29,7 +27,7 @@ extension ItemType {
             return "NSTouchBarItem.Identifier.flexibleSpace"
         }
     }
-
```
Comment (ReDetection) [inline MTMR/TouchBarController.swift]: Hn. Does my Xcode leave a lot of spaces here and your's doesn't?

**Pull request merged**